ReactWindows/ReactNative.Shared/UIManager/PropertySetter.cs

```diff
@@ -95,6 +95,16 @@ public void UpdateViewManagerProperty(IViewManager viewManager, DependencyObject
             Invoke(viewManager, GetViewManagerArgs(view, props));
         }
 
+        public void UpdateViewManagerExtensionProperty(IViewManagerExtension extension, DependencyObject view, ReactStylesDiffMap props)
+        {
```

Comment:
same here -- can we use the TFrameworkElement parameterized type pattern established elsewhere?